tests/test_sequence.py

```diff
@@ -1729,10 +1729,16 @@ def test_sequence(reg, device, patch_plt_show):
     assert str(seq) == str(seq_)
 
 
+@pytest.mark.parametrize("custom_phase_jump_time", (None, 0))
 @pytest.mark.parametrize("eom", [False, True])
-def test_estimate_added_delay(eom):
+def test_estimate_added_delay(eom, custom_phase_jump_time):
+    ryd_ch_obj = dataclasses.replace(
+        AnalogDevice.channels["rydberg_global"],
+        custom_phase_jump_time=custom_phase_jump_time,
+    )
+    device = dataclasses.replace(AnalogDevice, channel_objects=(ryd_ch_obj,))
     reg = Register.square(2, 5)
-    seq = Sequence(reg, AnalogDevice)
+    seq = Sequence(reg, device)
```

Comment:
I am wondering if this is hyper user-friendly... I don't know if any users will modify the channel in a Device to define their custom phase_jump_time (it will be a good test to see if people understand the difference between a backend and a device though)

Reply:
This change is more for us, so that we can override the default phase jump time on our devices.